pkg/controller/v1alpha2/applicationconfiguration/applicationconfiguration.go

```diff
@@ -346,10 +345,36 @@ func (r *OAMApplicationReconciler) updateStatus(ctx context.Context, ac *v1alpha
 		}
 	}
 	ac.Status.Workloads = append(ac.Status.Workloads, revisionStatus...)
-
+	// patch the extra fields in the status that is wiped by the Status() function
```

Comment:
`Status()` is not specific enough to pinpoint the code location.
Better point out its parent function/object as well.